discord/types/appinfo.py

```diff
@@ -78,3 +79,7 @@ class PartialAppInfo(BaseAppInfo, total=False):
 class GatewayAppInfo(TypedDict):
     id: Snowflake
     flags: int
+
+
+class ListAppEmojis(TypedDict):
+    items: list[Emoji]
```

Comment:
Drops support for Python 3.8
```suggestion
    items: List[Emoji]
```